### PR TITLE
[MIG][10.0] website_form_recaptcha

### PR DESCRIPTION
**Pull request by lasley**
This was theoretically as simple as changing the version numbers. I would like a functional test though, so WIP until my PR on website_crm_recaptcha so that we may confirm the base works.
- Update website_form_recaptcha to v10

### CODE REVIEW

**Comment by lasley** [comment]
This has been fixed & is ready for review.

Functional review can take place in #268

**Review by pedrobaeza — APPROVED** [review]

**Comment by lasley** [comment]
@tedsalmon please review

**Review by tedsalmon — APPROVED** [review]
👍  LGTM!

File: website_form_recaptcha/static/src/js/field_recaptcha.js [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
+
+        start: function() {
+            var self = this;
+            this._super();
+            this.$captchas = self.$('.o_website_form_recaptcha');
+            $.ajax({
+                url: '/website/recaptcha/',
+                method: 'GET',
+                success: function(data){
+                data = JSON.parse(data);
```
Comment (simahawk) [inline website_form_recaptcha/static/src/js/field_recaptcha.js]: bad indentation here ;)

**Review by simahawk — APPROVED** [review]
LGTM despite a small issue w/ JS indentation

**Pull request merged**